grails-app/controllers/com/unifina/controller/api/IntegrationKeyApiController.groovy

```diff
@@ -12,6 +13,13 @@ import groovy.json.JsonSlurper
 
 @Secured(["IS_AUTHENTICATED_ANONYMOUSLY"])
 class IntegrationKeyApiController {
+	static allowedMethods = [
```

Comment:
Seems like duplication of what you have specified in `UrlMappings.groovy`. Keep this one and remove the method definitions from `UrlMappings`?